app/assets/stylesheets/adopt.css.scss

```diff
@@ -35,3 +36,55 @@
     font-weight: normal;
   }
 }
+
+.list-of-rats {
+  list-style-type: none;
+  margin: 0;
+  padding: 0;
+  text-align: center;
+  li{
+    display: inline-block;
+    height: 150px;
+    margin: 100px 1em 1em 0;
+    position: relative;
+    width: 150px;
+    &:hover span.text-content {
```

Comment:
Rule declaration should be preceded by an empty line<br>Avoid qualifying class selectors with an element.<br>Selector should have depth of applicability no greater than 2, but was 3